app/models/institution_builder.rb

```diff
@@ -117,49 +125,39 @@ def self.add_accreditation(version_number)
     # Set the accreditation_type according to the hierarchy hybrid < national < regional
     # We include only those accreditation that are institutional and currently active.
     str = <<-SQL
-      UPDATE institutions SET accreditation_type = CASE
-        WHEN at_types @> '{ regional }' THEN 'regional'
-        WHEN at_types @> '{ national }' THEN 'national'
-        WHEN at_types @> '{ hybrid }' THEN 'hybrid'
-        ELSE NULL
-      END
-      FROM (
-        SELECT "cross", array_agg(DISTINCT(accreditation_type)) AS at_types
-        FROM accreditations
-        WHERE "cross" IS NOT NULL
-          AND accreditation_type IS NOT NULL
-          AND periods LIKE '%current%'
-          AND csv_accreditation_type = 'institutional'
-        GROUP BY "cross") AS cross_type_arr
-      WHERE cross_type_arr.cross = institutions.cross
-        AND institutions.version = #{version_number};
+      UPDATE institutions SET
+        accreditation_type = accreditations.accreditation_type
+      FROM accreditations
+      WHERE institutions.cross = accreditations.cross
+        AND accreditations.cross IS NOT NULL
+        AND accreditations.periods LIKE '%current%'
+        AND accreditations.csv_accreditation_type = 'institutional'
+        AND institutions.version = #{version_number}
+        AND accreditations.accreditation_type =
```

Comment:
do we have some performance metrics around this query vs what it replaces? This has a lot more conditional branches from the looks of it.